app/Notification/AppNotificationInterface.php

```diff
@@ -4,12 +4,12 @@
 namespace App\Notification;
 
 
-use App\Notification\Client\HTTPClientAdapterInterface;
-use App\Notification\Client\ResponseAdapterInterface;
+use App\Notification\Client\HttpClientAdapterInterface;
```

Comment:
This namespace doesn't match the file structure.

```suggestion
use App\Notification\HttpClientAdapterInterface;
```